ecms_base/modules/custom/ecms_migration/src/Plugin/migrate/process/RemoveAttribute.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\ecms_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateException;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Uses a regex to remove the given attribute.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "remove_attribute"
+ * )
+ *
+ * Remove 'style' attributes from HTML tags:
+ * @code
+ * field_text:
+ *   plugin: remove_attribute
+ *   source: text
+ *   attribute: 'style'
+ * @endcode
+ */
+class RemoveAttribute extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
```

Comment:
Typehints on the parameters and the return value, or will that violate the `ProcessPluginBase` method override?

Reply:
Nope I think we're good to add!